Babylon/commands/api/scenarioruns/cumulated_logs.py

```diff
@@ -0,0 +1,38 @@
+from logging import getLogger
+from typing import Any
+
+from click import command, option
+
+from Babylon.commands.api.scenarioruns.service.api import ScenarioRunService
+from Babylon.utils.credentials import pass_azure_token
+from Babylon.utils.decorators import timing_decorator, wrapcontext, retrieve_state
+from Babylon.utils.response import CommandResponse
+
+logger = getLogger("Babylon")
+
+
+@command()
+@wrapcontext()
+@timing_decorator
+@pass_azure_token("csm_api")
+@option("--organization-id", "organization_id", type=str)
+@option("--scenariorun-id", "scenariorun_id", type=str)
```

Comment:
just a question : do we use snake case or kebab case for options ?

Reply:
dans la convention de nommage pour les options ou flags de CLI  est utiliser le (-) comme separateur et le (--) au début